website/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Transport API Repository coverage map</title>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel='stylesheet' href='https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.css' />
+    <script src='https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.js'></script>
```

Comment:
Let's make sure we don't accidentally inherit breaking changes.

```suggestion
    <script src='https://unpkg.com/maplibre-gl@5/dist/maplibre-gl.js'></script>
```

Same for the CSS above.